OrangeHRMTestFramework/Tests/OrangeHRM/UserManagementTests.cs

```diff
@@ -0,0 +1,225 @@
+﻿using NUnit.Framework;
+using OrangeHRMTestFramework.Data.Constants;
+using OrangeHRMTestFramework.Data.Enums;
+using OrangeHRMTestFramework.Helpers;
+using OrangeHRMTestFramework.Models;
+using OrangeHRMTestFramework.PageObjects.OrangeHRM;
+using OrangeHRMTestFramework.PageObjects.OrangeHRM.DataGrids;
+using OrangeHRMTestFramework.PageObjects.OrangeHRM.Filters;
+using OrangeHRMTestFramework.PageObjects.OrangeHRM.Forms;
+using OrangeHRMTestFramework.PageObjects.OrangeHRM.Popups;
+
+namespace OrangeHRMTestFramework.Tests.OrangeHRM
+{
+    public class UserManagementTests : BaseTest
```

Comment:
Please run this suite on your side. Looks like tests are not stable and sometimes fail

Reply:
Made changes for sort by desc test (added pagination) and changed locator for action buttons on the data grid. Seems to be working better.